src/main/java/classify/commands/AddStudent.java

```diff
@@ -69,6 +69,10 @@ public static void addStudent(ArrayList<Student> masterStudentList, Scanner in,
                     "please restore the student instead.");
             UI.printDivider();
             return;
+        } catch (NullPointerException e) {
```

Comment:
Good catch
![Thumbsup21-ezgif com-webp-to-gif-converter](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/110646458/e7aebb65-7f57-4ecd-a32f-02497352ac17)